ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -657,22 +678,103 @@ public void shouldInsertInto() throws Exception {
   }
 
   @Test
-  public void shouldHandleErrorResponseFromInsertInto() {
```

Comment:
This is important test coverage (that error responses are properly exposed to the user). If we are removing the specific cause of the error used in the existing test, then let's replace the error with a different error thrown from the endpoint, rather than removing the test entirely.

Reply:
As mentioned above, I can create separate PR to add more tests catching different exceptions

Reply:
As above, I don't think we should be removing test coverage in this PR. Let me know if what I'm saying isn't making sense, I'm happy to sync offline.